### DISCUSSION TITLE
May I set rtsp session timeout of the rtsp-server ?

### DISCUSSION THREAD

**Discussion by strong0716** [bluenviron/mediamtx#1555] [Questions]
If I use VLC as a rtsp server, there is a config that can set rtsp session time to zero
if I do this, the client keep connected even though client do not sent the rtsp alive signal to server
does the rtsp-simple-server can do this function ?

I use webrtc-streamer (another github project) to connect the rtsp-simple-server
and always reconnect every 60 sec interval

How can I avoid it ?

thank you very much <(_ _)>

**Comment by aler9** [comment]
Hello, your RTSP client (webrtc-streamer) have to periodically send keepalive packets to the server in order to keep the RTSP session open. That's how RTSP works. Otherwise, sessions would be kept open forever, resulting in RAM exhaustion.

Since the RTSP client is an open source project, contact their authors in order to implement the keepalive feature.

[a comment by an automated account is not shown]
